### ISSUE TITLE
Change TimeUpdated interval

### ISSUE THREAD

**Issue by thumaman** [Megabit/Blazorise#5465]
Hi, is there a way of setting the interval for how many times per second TimeUpdated is called in your Video Player?

<Video Source="@videoSource" TimeUpdate="HandleTimeUpdate" />

**Comment by stsrki** [comment]
No, it is not possible. We propagate all the events automatically from the player. 

However, it is not a bad idea to have some kind of throttle or debouncing for internal events. We might look into it in the future. Or if you happen to contribute it might come faster into the production.

**Comment by thumaman** [comment]
@David-Moreira great recommendation, thank you!
